### PR TITLE
Feature to main - portfolio Alex

### PR DESCRIPTION
**Pull request by SniffleSneeze**
I do know that I need to update my About me description but I figure I would already ask you your feedback on the work that has been done !!

### CODE REVIEW

File: aboutme.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!Doctype html>
+<html lang="en" ontouchmove>
+	<head>
+		<title>Portfolio ALex F.</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link href="./css/normalise.css" type="text/css" rel="stylesheet">
+		<link href="./css/style.css" type="text/css" rel="stylesheet">
+
```
Comment (mporam) [inline aboutme.html]: empty line?

File: aboutme.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<!Doctype html>
+<html lang="en" ontouchmove>
+	<head>
+		<title>Portfolio ALex F.</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link href="./css/normalise.css" type="text/css" rel="stylesheet">
```
Comment (mporam) [inline aboutme.html]: you dont need to use `./` in the href. just doing `css/normalise.css` will work fine

File: aboutme.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+		<link href="./css/style.css" type="text/css" rel="stylesheet">
+		
+	</head>
+	
+	<header>
```
Comment (mporam) [inline aboutme.html]: all HTML that is displayed on the page should go *inside* the `<body>` tag

File: aboutme.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+		<main id="aboutme-body-image">
+			
+			<div class="astronaute">
+				<img src="./images-logo/astronaute.png" alt="image of an Astronaute">
+			</div>
+			
+			<div class="content-box aboutme">
+				<h2>-About Me-</h2>
+				<div class="aboutme-div">
+					<p>Hey Welcome to my first website !<br>
```
Comment (mporam) [inline aboutme.html]: generally its better to use 2 `<p>` tags than 1 with a `<br>`

File: aboutme.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+	<body class="body-image">
+		<main id="aboutme-body-image">
+			
+			<div class="astronaute">
+				<img src="./images-logo/astronaute.png" alt="image of an Astronaute">
+			</div>
+			
+			<div class="content-box aboutme">
+				<h2>-About Me-</h2>
+				<div class="aboutme-div">
```
Comment (mporam) [inline aboutme.html]: `aboutme-div` isnt a very semantic class name. what if you want to put it on something that isn't a div?

File: contact.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+				<a href="./index.html" class="navlink">Home</a>
+				<a href="./portfolio.html"class="navlink">Portfolio</a>
+				<a href="./aboutme.html"class="navlink">About me</a>
+				<a href="./contact.html"class="navlink current">Contact</a>
+			</div>
+			
+		</nav>
+	</header>
+	
+	<body onclick class="body-image">
```
Comment (mporam) [inline contact.html]: `onclick` ?

File: contact.html [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+				
+				<div class="container">
+					<a href="https://www.linkedin.com/in/alex-ferreira-de-sousa-a9a746a4/"><img src="./images-logo/LI-In-Bug.png" alt="Linkedin logo" class="logo"></a>
+					<a href="https://github.com/SniffleSneeze"><img src="./images-logo/GitHub-Mark-Light-120px-plus.png" alt="GitHub logo" class="logo"></a>
+					<a href="mailto:info.alexFerreira@gmail.com""><img src="./images-logo/email.png" alt="Email"></a>
+				</div>
+			</div>
+		</main>
+	</body>
+
```
Comment (mporam) [inline contact.html]: 2 empty lines? 1 would be okay, 2 is too many

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,281 @@
+* {
+	box-sizing:border-box;
```
Comment (mporam) [inline css/style.css]: missing spaces after the `:`s

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,281 @@
...
+	font-family:arcadeclassic;
+	color:white;
```
Comment (mporam) [inline css/style.css]: colors should always be in hex please

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,281 @@
...
+	cursor:url("../images-logo/cursor-spaceship-medium.png"),auto;
+}
+
+@font-face {
+  font-family: arcadeclassic;
+  src: url("../arcadeclassic/arcadeclassic.ttf");
+}
+
+body {
+
```
Comment (mporam) [inline css/style.css]: empty line?

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,281 @@
...
+	
+	background-image:url("../images-logo/planet2.png");
+	background-size: 2200px 1122px;
+	background-repeat: no-repeat;
+	background-blend-mode: multiply;
+	
+}
+
+.current {
+	border-top:2px solid white;
```
Comment (mporam) [inline css/style.css]: missing spaces after the `:` and colors should be in hex

File: css/style.css [hunk 12]
```diff
@@ -0,0 +1,281 @@
...
+	margin-left: auto;
+	margin-right:auto;
+}
+
+.link {
+	display:flex;
+	justify-content:flex-end;
+}
+
+/*------------------Navbar start here------------------*/
```
Comment (mporam) [inline css/style.css]: seems unnecessary, the class is literally called navbar

File: css/style.css [hunk 13]
```diff
@@ -0,0 +1,281 @@
...
+.current {
+	border-top:2px solid white;
+	border-bottom:2px solid white;
+}
+
+.container {
+	display:flex;
+	justify-content:space-between;
+	width:80%;
+	margin-left: auto;
```
Comment (mporam) [inline css/style.css]: you can do this in less code: `margin: 0 auto;`

File: css/style.css [hunk 14]
```diff
@@ -0,0 +1,281 @@
...
+.link {
+	display:flex;
+	justify-content:flex-end;
+}
+
+/*------------------Navbar start here------------------*/
+
+.navbar {
+	margin-top: 20px;
+	background-color:none;
```
Comment (mporam) [inline css/style.css]: do you need to say no background color? that's default

File: css/style.css [hunk 15]
```diff
@@ -0,0 +1,281 @@
...
+	margin-top:auto;
+	margin-bottom:auto;
+	margin-right:10px;
+	padding-top:5px;
+	width: 6rem;
+	height: 2rem;
+	font-size:1rem;
+	text-align:center;
+	text-decoration:none;
+
```
Comment (mporam) [inline css/style.css]: empty line

File: css/style.css [hunk 16]
```diff
@@ -0,0 +1,281 @@
...
+	height:2px;
+	background-color:white;
+	transition: width .4s
+}
+
+a:hover::after {
+	width:100%;
+	transition: width .4s;
+}
+/*------------------Body Main start here------------------*/
```
Comment (mporam) [inline css/style.css]: comment seems unnecessary

File: css/style.css [hunk 17]
```diff
@@ -0,0 +1,281 @@
...
+main {
+	position:relative;
+}
+
+.content-box {
+	width:29rem;
+	margin:auto auto;
+	padding-top:10px;
+    text-align: center;
```
Comment (mporam) [inline css/style.css]: indentation is out

File: css/style.css [hunk 18]
```diff
@@ -0,0 +1,281 @@
...
+	font-size:1.4rem;
+}
+/*-----------------------HOME / INDEX SECTION---------------------*/
+
+.border {
+	border: 10px solid transparent;
+	border-image: url("../images-logo/border2.png") 10% stretch;	
+}
+
+#index-body-image {
```
Comment (mporam) [inline css/style.css]: why an ID?

File: css/style.css [hunk 19]
```diff
@@ -0,0 +1,281 @@
...
+/*-----------------------PORTFOLIO SECTION---------------------*/
+
+.planet-projet {
+	margin-top:50px;
+	padding-left:auto;
+	padding-right:auto;
+	position: relative;
+	height: 100%;
+	width:40%;
+	/*overflow:hidden;*/
```
Comment (mporam) [inline css/style.css]: if you dont need this, remove it from the code

File: css/style.css [hunk 20]
```diff
@@ -0,0 +1,281 @@
...
+	width:108px;
+}
+
+.project-p {
+	border-top: 2px solid white;
+	border-bottom: 2px solid white;
+	width:70%;
+	text-align:center;
+	position:absolute;
+	top: 0px;
```
Comment (mporam) [inline css/style.css]: dont need units on `0` values. 0 is 0 in any unit

File: css/style.css [hunk 21]
```diff
@@ -0,0 +1,281 @@
...
+	border-image: url("../images-logo/border2.png") 10% stretch;
+}
+
+#projet-left {
+	text-align:left;
+	padding-left:5px;
+	position:relative;
+	left:2%;
+}
+
```
Comment (mporam) [inline css/style.css]: extra line break

File: css/style.css [hunk 22]
```diff
@@ -0,0 +1,281 @@
...
+
+#projet-right {
```
Comment (mporam) [inline css/style.css]: why an id?

File: css/style.css [hunk 23]
```diff
@@ -0,0 +1,281 @@
...
+	border-radius: 15px;
+}
+
+.planet-projet img {
+	width:7rem;
+	border:8px solid transparent;
+	border-image: url("../images-logo/border2.png") 10% stretch;
+}
+
+#projet-left {
```
Comment (mporam) [inline css/style.css]: why an id?

File: css/style.css [hunk 24]
```diff
@@ -0,0 +1,281 @@
...
+
+
+#projet-right {
+	text-align:left;
+	padding-left:5px;
+	position:relative;
+	left:50%;
+}
+
+/*.planet-projet:hover {
```
Comment (mporam) [inline css/style.css]: please remove

File: css/style.css [hunk 25]
```diff
@@ -0,0 +1,281 @@
...
+	position:absolute;
+	top:75%;
+	left:50%;
+}
+
+.aboutme {
+	padding:0 0;
+	border: 10px solid transparent;
+	border-image: url("../images-logo/border2.png") 10% stretch;
+
```
Comment (mporam) [inline css/style.css]: empty lines

File: css/style.css [hunk 26]
```diff
@@ -0,0 +1,281 @@
...
+	overflow: auto;
+	height:70%;
+}
+
+.aboutme-div {
+	position: relative;
+	
+}
+
+/*.aboutme:hover {
```
Comment (mporam) [inline css/style.css]: remove

File: css/style.css [hunk 27]
```diff
@@ -0,0 +1,281 @@
...
+	animation: dropdown 1.3s ease-in forwards;
+}*/
+
+/*-----------------------CONTACT ME SECTION---------------------*/
+
+#contact-box-title {
+	font-size:3rem;
+	width:27rem;
+	margin:auto auto;
+    text-align: center;
```
Comment (mporam) [inline css/style.css]: incorrect indentation

File: css/style.css [hunk 28]
```diff
@@ -0,0 +1,281 @@
...
+	height:46px;
+}
+
+@keyframes dropdown {
+	100% {
+		height:30rem;
+	}
+}
+
+@media only screen and (max-width: 680px){
```
Comment (mporam) [inline css/style.css]: missing space between `)` and `{`

File: css/style.css [hunk 29]
```diff
@@ -0,0 +1,281 @@
...
+
+	#projet-right {
+		text-align:left;
+		padding-left:15px;
+		position:relative;
+		left:0%;
+	}
+	
+
+/*------------------------ ABOUT ME PAGE ----------------------------------*/
```
Comment (mporam) [inline css/style.css]: remove

File: css/style.css [hunk 30]
```diff
@@ -0,0 +1,281 @@
...
+	
+	
+
+/*------------------------ CONTACT PAGE ----------------------------------*/
```
Comment (mporam) [inline css/style.css]: remove

File: index.html [hunk 31]
```diff
@@ -0,0 +1,40 @@
+<!Doctype html>
+<html lang="en" ontouchmove>
```
Comment (mporam) [inline index.html]: `ontouchmove`?

File: index.html [hunk 32]
```diff
@@ -0,0 +1,40 @@
...
+		<link href="./css/normalise.css" type="text/css" rel="stylesheet">
+		<link href="./css/style.css" type="text/css" rel="stylesheet">
+		
+	</head>
+	
+	<header>
+		<nav class="container navbar">
+			<h1>Alex Ferreira</h1>
+			<div class="link">
+				<a href="./index.html" class="navlink current">Home</a>
```
Comment (mporam) [inline index.html]: dont need the `./`

File: index.html [hunk 33]
```diff
@@ -0,0 +1,40 @@
...
+			</div>
+			
+		</nav>
+	</header>
+	
+	<body class="body-image">
+		<main id="index-body-image">
+			<div class="content-box border">
+				<h2>- Alex Ferreira Webpage -</h2>
+				<p > Welcome on my Webpage ! have a look around if you have any question, go to the contact tab, I will be please to hear from you</p>
```
Comment (mporam) [inline index.html]: extra space inside the p tag?

File: index.html [hunk 34]
```diff
@@ -0,0 +1,40 @@
...
+			</div>
+		</main>
+	</body>
+	
+	
+	<footer class="container">
```
Comment (mporam) [inline index.html]: footer needs to be inside the body tag

File: portfolio.html [hunk 35]
```diff
@@ -0,0 +1,70 @@
...
+		<nav class="container navbar"> <!--Navbar start-->
+			<h1>Alex Ferreira</h1>
+			<div class="link">
+				<a href="./index.html" class="navlink">Home</a>
+				<a href="./portfolio.html"class="navlink current">Portfolio</a>
+				<a href="./aboutme.html"class="navlink">About me</a>
+				<a href="./contact.html"class="navlink">Contact</a>
+			</div>
+			
+		</nav> <!--Navbar end-->
```
Comment (mporam) [inline portfolio.html]: unnecessary html comments

File: portfolio.html [hunk 36]
```diff
@@ -0,0 +1,70 @@
...
+	</header>
+	
+	<body class="body-image"> <!--body start-->
```
Comment (mporam) [inline portfolio.html]: unnecessary comment

File: portfolio.html [hunk 37]
```diff
@@ -0,0 +1,70 @@
...
+			</div>
+			
+			<div class="planet-projet" id="projet-right" id="p2">
+				<h3>Project Nam</h3>
+				<img src="./images-logo/Lava.png" alt="Projet 1">
+				<p class="project-p"> "Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit..."
+"There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain..."</p>
+			</div>
+			
+			<div class="planet-projet" id="projet-left" id="p3">
```
Comment (mporam) [inline portfolio.html]: why are you using an id? you cant use the same ID more than once on a page, so technically this is invalid HTML. please use classes

**Review by mporam — CHANGES_REQUESTED** [review]

File: css/style.css [hunk 38]
```diff
@@ -0,0 +1,281 @@
+* {
+	box-sizing:border-box;
+	font-family:arcadeclassic;
+	color:white;
+	cursor:url("../images-logo/cursor-spaceship-medium.png"),auto;
+}
+
+@font-face {
+  font-family: arcadeclassic;
```
Comment (nealioa) [inline css/style.css]: Indent is inconsistent with the other rules.

File: css/style.css [hunk 39]
```diff
@@ -0,0 +1,281 @@
...
+	display:flex;
+	justify-content:flex-end;
+}
+
+/*------------------Navbar start here------------------*/
+
+.navbar {
+	margin-top: 20px;
+	background-color:none;
+}
```
Comment (nealioa) [inline css/style.css:39]: Have an empty line after the `}`.

File: css/style.css [hunk 40]
```diff
@@ -0,0 +1,281 @@
...
+	padding-top:5px;
+	width: 6rem;
+	height: 2rem;
+	font-size:1rem;
+	text-align:center;
+	text-decoration:none;
+	
+}
+
+a:active{
```
Comment (nealioa) [inline css/style.css]: Missing a space before `{`.

File: css/style.css [hunk 41]
```diff
@@ -0,0 +1,281 @@
...
+	padding-left:auto;
+	padding-right:auto;
+	position: relative;
+	height: 100%;
+	width:40%;
+	/*overflow:hidden;*/
+	transition-timing-function: ease-in;
+	transition-duration: 1s;
+	transition-property: width;
+
```
Comment (nealioa) [inline css/style.css]: Empty line.

File: index.html [hunk 42]
```diff
@@ -0,0 +1,40 @@
+<!Doctype html>
+<html lang="en" ontouchmove>
+	<head>
+		<title>Portfolio ALex F.</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link href="./css/normalise.css" type="text/css" rel="stylesheet">
+		<link href="./css/style.css" type="text/css" rel="stylesheet">
+
```
Comment (nealioa) [inline index.html]: Unnecessary empty line.

File: portfolio.html [hunk 43]
```diff
@@ -0,0 +1,70 @@
...
+	</header>
+	
+	<body class="body-image"> <!--body start-->
+		
+		<main id="portfolio-body-image"> <!--main start-->
+			<div class="content-box">
+				<h2>- Portfolio -</h2>
+			</div>
+			
+			<div class="planet-projet" id="projet-left" id="p1">
```
Comment (nealioa) [inline portfolio.html]: Two `id`s?
See Mike's comment below.

File: portfolio.html [hunk 44]
```diff
@@ -0,0 +1,70 @@
...
+"There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain..." </p>
+			</div>
+		</main> <!--main end-->
+	
+	</body> <!--body end-->
+	
+	
+	<footer class="container">
+		<div>
+			<p>&copy 2021 Alexander Ferreria</p>
```
Comment (nealioa) [inline portfolio.html]: HTML entities should end with a semicolon - `&copy;`

**Review by nealioa — CHANGES_REQUESTED** [review]

File: aboutme.html [hunk 45]
```diff
@@ -1,48 +1,42 @@
...
-	
 	<body class="body-image">
-		<main id="aboutme-body-image">
-			
+		<header>
+			<nav class="container navbar">
+				<h1>Alex Ferreira</h1>
+				<div class="link">
+					<a href="index.html" class="navlink">Home</a>
+					<a href="portfolio.html"class="navlink ">Portfolio</a>
```
Comment (mporam) [inline aboutme.html]: missing spaces before the class property

File: css/style.css [hunk 46]
```diff
@@ -1,281 +1,230 @@
...
 }
 
 .navlink {
-	display:inline-block;
-	margin-top:auto;
-	margin-bottom:auto;
-	margin-right:10px;
-	padding-top:5px;
+	display: inline-block;
+	margin: auto auto;
```
Comment (mporam) [inline css/style.css]: this can just be `margin: auto;`

File: css/style.css [hunk 47]
```diff
@@ -1,281 +1,230 @@
...
+	width: 100%;
 	transition: width .4s;
 }
-/*------------------Body Main start here------------------*/
+
 main {
-	position:relative;
+	position: relative;
+	height: auto;
+
```
Comment (mporam) [inline css/style.css]: remove this extra line break

File: css/style.css [hunk 48]
```diff
@@ -1,281 +1,230 @@
...
-	font-size:1.4rem;
+	letter-spacing: 0.11rem;
+	font-size: 1.4rem;
 }
 /*-----------------------HOME / INDEX SECTION---------------------*/
-
 .border {
 	border: 10px solid transparent;
 	border-image: url("../images-logo/border2.png") 10% stretch;	
+	background-color: #00000090;
```
Comment (mporam) [inline css/style.css]: use rgba for transparency, 8 digit hex doesn't work in all browsers:
https://caniuse.com/css-rrggbbaa

File: css/style.css [hunk 49]
```diff
@@ -1,281 +1,230 @@
...
+	margin-bottom:20px;
```
Comment (mporam) [inline css/style.css]: missing space after `:`

File: css/style.css [hunk 50]
```diff
@@ -1,281 +1,230 @@
...
-	#projet-left {
-		text-align:left;
-		padding-left:15px;
-		position:relative;
-		left:0%;
+	.projet-left {
+		text-align: left;
+		padding-left: 15px;
+		position: relative;
+		left: 0%;
```
Comment (mporam) [inline css/style.css]: dont need `%` on 0 values

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]

**Pull request merged**